src/main/java/com/backbase/ct/bbfuel/client/user/UserPresentationRestClient.java

```diff
@@ -46,4 +58,38 @@ public UserGetResponseBody getUserByExternalId(String userExternalId) {
             .as(UserGetResponseBody.class);
     }
 
+    public void createIdentityUserAndLogResponse(UsersPostRequestBody user){
+
+        Response response = createIdentity(user);
+
+
+        if (isBadRequestException(response, "User already exists")) {
```

Comment:
Will this message always be "User already exists"?

Reply:
I think it might be safer to use the error key if we return any.

Reply:
I don't think the relevant keys are returned, however the methods have been added in case that changes in the future